tokens/_aliases.yml

```diff
@@ -191,17 +192,17 @@ aliases:
   screen-xs: '320px' # content-xs + 16px
   screen-sm: '480px' # content-sm + 16px
   screen-md: '768px' # content-md + 32px
-  screen-lg: '1056px' # content-lg + 64px
+  screen-lg: '1024px' # content-lg + 32px
```

Comment:
What do the 16/32/64px addition represent?

Reply:
It's just the difference between the screen sizes and content sizes, to keep some space between the edge of the viewport and the content at each breakpoint. We could also say `$content-lg` is `$screen-lg - 32px`

Reply:
I think we should flip it in the docs. Since the screen sizes are the immutable numbers.

Reply:
that makes sense... let's do that when we update content sizes